src/ingest_validation_tools/table-schemas/assays/gcms-v1.yaml

```diff
@@ -0,0 +1,116 @@
+fields:
+# include: ../includes/vA/all_assays.yaml
+- name: assay_category
+  constraints:
+    enum:
+      - mass_spectrometry_imaging
+- name: assay_type
+  constraints:
+    enum:
+      - GC-MS
+- name: analyte_class
+  constraints:
+    enum:
+      # TODO: Definitely not protein?
+      - metabolites
+      - lipids
+# include: ../includes/v0/is_targeted.yaml
+
+# Level 2:
+
+# include: ../includes/v0/acquisition_instrument.yaml
+- name: ms_source
+  description: The ion source type used for surface sampling (MALDI, MALDI-2, DESI, or SIMS) or LC-MS/MS data acquisition (nESI)
```

Comment:
@cebriggs7135 -- In general, for enums, I'd prefer to avoid duplicating the enum in the description.